tests/consumers/acero_consumer.py

```diff
@@ -0,0 +1,52 @@
+import pyarrow as pa
+from pyarrow.lib import Table
+
+from .consumer import AbstractConsumer
+
+try:
+    import pyarrow.substrait as substrait
+except ImportError:
+    substrait = None
+
+
+class AceroConsumer(AbstractConsumer):
```

Comment:
We chatted offline -- I'm not sure I see the value in the abstract base class at this stage.

Reply:
Removed the abstract base class